Gordon360/Services/RecIM/TeamService.cs

```diff
@@ -344,8 +349,47 @@ public async Task<TeamViewModel> UpdateTeamAsync(int teamID, TeamPatchViewModel
 
             return t;
         }
+
+        private async Task SendInviteEmail(int teamID, string inviteeUsername, string inviterUsername)
+        {
+            var team = _context.Team.FirstOrDefault(t => t.ID == teamID);
+            var activity = _context.Activity.FirstOrDefault(a => a.ID == team.ActivityID);
+            var invitee = _accountService.GetAccountByUsername(inviteeUsername);
+            var inviter = _accountService.GetAccountByUsername(inviterUsername);
+
+            string from_email = _config["Emails:Sender:Username"];
+            string to_email = invitee.Email;
+            string messageBody =
+                 $"Hey {invitee.FirstName}!<br><br>" +
+                $"{inviter.FirstName} {inviter.LastName} has invited you join <b>{team.Name}</b> for <b>{activity.Name}</b> <br>" +
```

Comment:
Is this a notification for an invitation or a notification that you have been added to a team. I think the language can be a little bit confusing

Reply:
In short, there is gaps in our workflow because we still have use cases to handle
This is the proposed workflow that our customer would like:
1) User/participant can be in Rec-IM OR NOT (as in waiver not signed)
2) User/participant will be emailed when they get invited to a team (not joined)
3) Invited User/participant goes to /recim and has the option of declining/accepting team invite
4) If the User/participant IS NOT cleared (no waiver signed), they will be prompted to sign the waiver